From 70683313424b73542f0f2b4c8432785409c12f1e Mon Sep 17 00:00:00 2001
From: Will Thompson <wjt@endlessos.org>
Date: Fri, 29 Apr 2022 11:20:35 +0100
Subject: [PATCH 08/13] metainfo: Use imperative voice in summary
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Tobias Bernard writes in “Get your apps ready for Software 41”
<https://blogs.gnome.org/tbernard/2021/09/07/ready-for-software-41/>:

> Writing a good summary
> ----------------------
>
> […]
>
> Some general guidance:
>
> - Keep it short (less than 35 characters)
> - Be engaging and make people curious
> - Use imperative if possible (e.g. “Browse the web” instead of “A web browser”)
> - Use sentence case

As it happens, the Comment from the .desktop file expresses the same
idea as the current summary and follows that guidance. It also already
has a number of translations (although not all of those are less than 35
characters).
---
 gobby-0.5.metainfo.xml.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gobby-0.5.metainfo.xml.in b/gobby-0.5.metainfo.xml.in
index 2eda036..e76ecb9 100644
--- a/gobby-0.5.metainfo.xml.in
+++ b/gobby-0.5.metainfo.xml.in
@@ -8,7 +8,7 @@
     <binary>gobby-0.5</binary>
   </provides>
   <_name>Gobby</_name>
-  <_summary>Collaborative text editor</_summary>
+  <_summary>Edit text files collaboratively</_summary>
 
   <description>
     <_p>
-- 
2.30.2

